stories/topics/con-gcp-pull-deploy-container-image.adoc

```diff
@@ -0,0 +1,36 @@
+[id="con-gcp-pull-deploy-container-image"]
+
+= Pulling the ansible-on-clouds-ops container image
+
+Pull the Docker image for the Ansible on Clouds operational container with the same tag as the version you are deploying to.
+
+[NOTE]
+====
+Before pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com.
```

Comment:
it should be `registry.redhat.io` in both places. Sorry forgot to uodate this in hackmd